src/app/frank/fd_frank_forward.c

```diff
@@ -98,6 +105,18 @@ run( fd_frank_args_t * args ) {
       then = now + (long)fd_tempo_async_reload( rng, async_min );
     }
 
+    ulong begin_version = fd_mvcc_begin_read( &leader_schedule->mvcc);
+    if( begin_version != version && begin_version % 2 == 0 ) {
```

Comment:
wouldn't this skip the very first update?

Reply:
Can use FD_LIKELY here to optimize the hot path.

Reply:
It doesn't skip the first update, I'll add the `FD_UNLIKELY`.